### PR TITLE
Use the dll from the plugin directory

### PR DESCRIPTION
**Pull request by JackeJR**
Simple edit to use the dll from plugin dir

### CODE REVIEW

File: plugin/vimtweak.vim [hunk 1]
```diff
@@ -2,7 +2,9 @@ if !has('gui_running') || (!has('win32') && !has('win64'))
   finish
 endif
 
-let g:vimtweak_dll_path = get(split(globpath(&rtp, has('win64') ? 'vimtweak64.dll' : 'vimtweak32.dll'), '\n'), 0, '')
+let g:vimtweak_dll_path = expand('<sfile>:p:h:h')
+let g:vimtweak_dll_path .= has('win64') ? '/vimtweak64.dll' : '/vimtweak86.dll'
```
Comment (mattn) [inline plugin/vimtweak.vim]:
```
let g:vimtweak_dll_path = expand('<sfile>:p:h:h') . (has('win64') ? '/vimtweak64.dll' : '/vimtweak86.dll')
```

File: plugin/vimtweak.vim [hunk 2]
```diff
@@ -2,8 +2,8 @@ if !has('gui_running') || (!has('win32') && !has('win64'))
   finish
 endif
 
-let g:vimtweak_dll_path = expand('<sfile>:p:h:h')
-let g:vimtweak_dll_path .= has('win64') ? '/vimtweak64.dll' : '/vimtweak86.dll'
+let g:vimtweak_dll_path = expand('<sfile>:p:h:h') . has('win64') ? '/vimtweak64.dll' : '/vimtweak86.dll'
```
Comment (mattn) [inline plugin/vimtweak.vim]: s/86/32/ ?

Comment (JackeJR) [inline plugin/vimtweak.vim]: yes my bad. copy and paste from another section.

**Comment by JackeJR** [comment]
by the way, there is a pure vimscript + python approach for:
https://gist.github.com/skywind3000/8eb41acd9d5175715694c765f92fa667

**Comment by JackeJR** [comment]
And I think it is possible to have a pure vimscript version since vimscript can call the user32.dll.

**Comment by mattn** [comment]
If you use python, it's NOT pure Vim script. It's just python.

**Comment by mattn** [comment]
Thank you.

**Comment by JackeJR** [comment]
yes, the linked version uses python. 

I was saying that the way the various functions are invoked is through user32.dll. It is therefore possible to code up a purely vimscript version without using python although the python version seems quite elegant. It may also just be my inexperience with vimscript.

**Pull request merged**